src/main/docs/guide/kubernetes-client-openapi.adoc

```diff
@@ -0,0 +1,52 @@
+The `micronaut-kubernetes-client-openapi` module uses Micronaut Netty HTTP Client and generated apis and modules from https://github.com/kubernetes-client/java/blob/master/kubernetes/api/openapi.yaml[Kubernetes Client OpenAPI Spec].
+
+First you need to add a dependency on the `micronaut-kubernetes-client-openapi` module:
+
+dependency:io.micronaut.kubernetes:micronaut-kubernetes-client-openapi[]
+
+Then you can simply use Micronaut injection to get configured apis object from package `io.micronaut.kubernetes.client.openapi.api`:
+
+[source,java]
```

Comment:
use the `snippet:` macro to include a real code snippet that is tested if possible

Reply:
Haven't done this yet. Will do.

Reply:
Done